src/workflow/workflow.c

```diff
@@ -45,7 +49,13 @@ void workflow_confirm_dismiss(const char* title, const char* body)
 
 void workflow_start(void)
 {
+#if PLATFORM_BITBOXBASE == 1
+    usart_start();
+    hww_setup();
+    usart_endpoint_enable(USART_ENDPOINT_HWW);
+#else
```

Comment:
I prefer to be explicit here. Use `#elif PLATFORM_BITBOX02` so that it doesn't work by default if we add a new platform in the future.

Reply:
Fixed